bin/install.sh

```diff
@@ -0,0 +1,129 @@
+#!/bin/bash
+
+# This script downloads the latest release version of EAS CLI from GitHub and
+# installs it in /usr/local/bin
+
+{
+  # All code is inside a block to ensure the script executes only when
+  # downloaded completely.
+
+  set -e
+
+  prefix="${EAS_PREFIX:-/usr/local}"
+  temp_dir="$(mktemp -d)"
+  
+  if  [ -t 1 ] && [ -z "$NO_COLOR" ]
+  then
+    # Use colors when stdout is a terminal and NO_COLOR isn't set.
+    text_bold=$'\033[1m'
+    text_normal=$'\033[22m'
+    text_red=$'\033[31m'
+    text_default_color=$'\033[39m'
+  else
+    # Disable colors
+    text_bold=""
+    text_normal=""
+    text_red=""
+    text_default_color=""
+  fi
+
+  abort() {
+    echo -e "$text_bold$text_red$1$text_normal$text_default_color"
+    exit 1
+  }
+
+  install() {
+    needs_sudo=""
+    # Make sure path exists and can be written to by the current user.
+    if ! mkdir -p "$prefix/lib" || [[ ! -w "$prefix/lib" ]]
+    then
+      needs_sudo=true
+    fi
+    if ! mkdir -p "$prefix/bin" || [[ ! -w "$prefix/bin" ]]
+    then
+      needs_sudo=true
+    fi
+    if ! rm -rf "$prefix/lib/eas"
+    then
+      needs_sudo=true
+    fi
+    # Run commands with sudo if necessary.
+    if [ -n "$needs_sudo" ]
+    then
+      echo "Installing requires superuser access."
+      echo "The sudo command will prompt for your password."
+      sudo rm -rf "$prefix/lib/eas"
+      sudo mv "$temp_dir/eas" "$prefix/lib/eas"
+      sudo ln -fs "$prefix/lib/eas/bin/eas" "$prefix/bin/eas"
```

Comment:
Should we chown lib/eas and bin/eas so they are owned by the current user (not superuser?)

Reply:
tar is preserving ownership base on UID/GID (in this case it's 1000). So if you run this script now lib/eas lib eas is owned by main user (in most cases), `bin/eas` is owned by root. 

Nothing in system directories that affects all users in the system should be ever owned by user, the only exception I have seen was users that exist for specific services( I think jenkins is doing sth like that).

> Should we chown lib/eas and bin/eas so they are owned by the current user (not superuser?)

Regardless if you agree with main previous point explicit `chown` is necessary to either `root` or current user, but I strongly believe that it should be owned by root if it's in `/usr`

Reply:
With Homebrew the system files are owned by one user and it generally works well because most people have their own computer.